samcli/local/lambdafn/runtime.py

```diff
@@ -250,7 +252,7 @@ def signal_handler(sig, frame):
         timer.start()
         return timer
 
-    def _get_code_dir(self, code_path):
+    def _get_code_dir(self, code_path: str) -> str:
         """
         Method to get a path to a directory where the Lambda function code is available. This directory will
```

Comment:
nit: update the docstring to tell that this is also used for Layer

Reply:
updated 👍